diskutil.c

```diff
@@ -77,6 +77,24 @@ static int get_io_ticks(struct disk_util *du, struct disk_util_stat *dus)
 	return ret != 10;
 }
 
+static uint64_t safe_32bit_diff(uint64_t nval, uint64_t oval)
+{
+	/* Linux kernel prints some of the stat fields as 32-bit integers. It is
+	 * possible that the value overflows, but since fio uses unsigned 64-bit
+	 * arithmetics in update_io_tick_disk(), it instead results in a huge
```

Comment:
Suggest changing from `arithmetics` to `arithmetic`, sorry not grammar police. :)